queries/highlights.scm

```diff
@@ -1,76 +1,104 @@
-; Comments
-(line_comment) @comment
+(comment) @comment
 
-(block_comment) @comment
+(ty (id)) @type
+
+(valid_semver) @property
+
+(world_item
+  name: (id) @property)
+
+(interface_item
+  name: (id) @property)
+
+(type_item
+  alias: (id) @property)
+
+(func_item
+  name: (id) @function)
+
+(handle
+  (id) @type)
+
+(named_type
+  name: (id) @variable.parameter)
+
+(record_item
+  name: (id) @variable)
+
+(record_field
```

Comment:
To comply with the [nvim-treesitter standards](https://github.com/nvim-treesitter/nvim-treesitter/blob/master/CONTRIBUTING.md#highlights), perhaps this should be `@variable.member`?

Reply:
should  be better now